cmd/link/userrole/cmd.go

```diff
@@ -79,9 +85,14 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 			reporter.Errorf("Failed to close OCM connection: %v", err)
 		}
 	}()
-	currentAccount, err := ocmClient.GetCurrentAccount()
-	if err != nil {
-		reporter.Errorf("Error getting current account: %v", err)
+
+	accountID := args.accountID
+	if accountID == "" {
+		currentAccount, err := ocmClient.GetCurrentAccount()
+		if err != nil {
+			reporter.Errorf("Error getting current account: %v", err)
+		}
+		accountID = currentAccount.ID()
```

Comment:
if the org admin running for the other user forgets to send the account ID we link their account ID . We are asking for confirmation is it enough or do we need further validation? 

.```
/rosa link user-role --role-arn arn:aws:iam::765374464689:role/test1-User-pvasanth_openshift-Role --account-id 1WDBDSHoNGbe8dMIV7mQYDttlOf
I: Linking User role
? Link the 'arn:aws:iam::765374464689:role/test1-User-pvasanth_openshift-Role' role with account '1WDBDSHoNGbe8dMIV7mQYDttlOf'? Yes
I: Successfully linked role-arn 'arn:aws:iam::765374464689:role/test1-User-pvasanth_openshift-Role' with account '1WDBDSHoNGbe8dMIV7mQYDttlOf'
```

Sample output of the  admin user running for the non org admin user